src/Microphysics1M.jl

```diff
@@ -113,11 +113,15 @@ function radar_reflectivity(
     ρ::FT,
 ) where {FT}
 
-    n0 = get_n0(pdf)
-    λ = lambda(pdf, mass, q, ρ)
-    Z₀ = FT(1e-18)
+    # change units for accuracy
+    n0 = get_n0(pdf) * FT(1e-12)
+    λ = lambda(pdf, mass, q, ρ) * FT(1e-3)
 
-    return 10 * log10((720 * n0 / λ^7) / Z₀)
+    Z = (λ == FT(0)) ? FT(0) : (720 * n0 / λ^7)
+    log_10_Z₀ = FT(-18)
+    log_Z = FT(10) * (log10(Z) - log_10_Z₀ - FT(9))
+
+    return (log_Z < FT(-300)) ? FT(-300) : log_Z
```

Comment:
Do we have to clip it here to -300? Would it be ok to just output very small numbers and deal with it when postprocessing the output.

Also `return max(FT(-300), log_z)` is maybe shorter?

Reply:
It needs a clipping because it could have -Inf values. We just have to choose how low we want the clipping value! + I'll change to `return max(FT(-300), log_z)`

Reply:
You can check the minimum meaningful number that log10 returns and set the threshold to that.